src/info/info.ts

```diff
@@ -186,7 +186,7 @@ port.onMessage.addListener(async (message) => {
             var firstCall: boolean = true;
 
             // get data from the handler
-            const imageSrc = rendering["data"]["image"] as string;
+            const imageSrc = rendering["data"]["graphic"] as string;
```

Comment:
Is it a conscious decision not to also rename the variables, e.g., `graphicSrc` for consistency?